ksqldb-udf/pom.xml

```diff
@@ -36,6 +36,16 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.errorprone</groupId>
+            <artifactId>error_prone_annotations</artifactId>
+            <version>2.2.0</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
```

Comment:
we can do this in a follow-up, but can you create a ticket to remove these dependencies as well? these common deps often cause conflicts when the projects that pull them in have their own dependencies on the libs

Reply:
I had removed these from pom but then it wasn't compiling. Tracked in this issue #5206